#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix Romania locale can't be built.
# DP: Related bugs: #245657
# DP: Dpatch author: GOTO Masanori <gotom@debian.org>
# DP: Patch author: Ulrich Drepper  <drepper@redhat.com>
# DP: Upstream status: In CVS
# DP: Status Details: 
# DP: Date: 2004-04-28

PATCHLEVEL=0

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p$PATCHLEVEL < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p$PATCHLEVEL < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
Index: locale/iso-3166.def
===================================================================
RCS file: /cvs/glibc/libc/locale/iso-3166.def,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- locale/iso-3166.def	7 Aug 2001 18:50:46 -0000	1.3
+++ locale/iso-3166.def	4 Dec 2003 04:33:59 -0000	1.4
@@ -180,7 +180,7 @@
 DEFINE_COUNTRY_CODE ("PUERTO RICO", PR, PRI, 630)
 DEFINE_COUNTRY_CODE ("QATAR", QA, QAT, 634)
 DEFINE_COUNTRY_CODE ("REUNION", RE, REU, 638)
-DEFINE_COUNTRY_CODE ("ROMANIA", RO, ROM, 642)
+DEFINE_COUNTRY_CODE ("ROMANIA", RO, ROU, 642)
 DEFINE_COUNTRY_CODE ("RUSSIAN FEDERATION", RU, RUS, 643)
 DEFINE_COUNTRY_CODE ("RWANDA", RW, RWA, 646)
 DEFINE_COUNTRY_CODE ("SAINT HELENA", SH, SHN, 654)
